06-release-engineering-analytics.Rmd

```diff
@@ -321,27 +324,122 @@ iteratively and planned for four weekly milestones.
 
 ## Answers
 
-<!-- TODO -->
+In this section, we will give an answer to each of the research questions
+that we have presented in [Section 7.1.1](#research-questions-1).
 
-### RQ1: ...
+### RQ1: Modern Release Engineering Practices
 
-<!-- TODO -->
+This section will give an answer to the question:
+_How is modern release engineering done in practice?_
+
+@adams2016a and @karvonen2017a have described release engineering
+practices that are currently in use in the industry.
+@adams2016a focused on modern release engineering,
+while @karvonen2017a investigated agile release engineering,
+which is a subset of all modern release engineering.
+Both studies agreed that modern release engineering consists
+of the following components:
+
+- **Rapid Releases (RR).**
+
+    In contrast with traditional release cycles,
+    RRs regularly push new releases to users in a regular schedule.
+    As an example, FireFox releases a new version every six weeks.
+
+- **Continuous Deployment or Continuous Delivery (CD).**
+
+    These two terms describe the ability to deploy (or deliver)
+    a software product to the end user automatically
+    whenever new code is committed or merged.
```

Comment:
"laukkanen2018a" nicely distinguishes between these terms:

> Continuous delivery (CD) (Humble and Farley 2010) is CI combined with the requirement that changes, after passing automated tests, can be deployed to production environment. However, production deployments are not necessarily done for each verified change.
Continuous deployment (CDep) (Fitz 2009) is CD combined with the requirement that verified changes are automatically deployed to production environment. 

Maybe we should also make this distinction